server/types/callbacks.go

```diff
@@ -28,6 +26,11 @@ type Callbacks interface {
 	//   non-zero value to indicate the rejection reason (typically 401, 429 or 503).
 	//   HTTPResponseHeader may be optionally set (e.g. "Retry-After: 30").
 	OnConnecting(request *http.Request) ConnectionResponse
```

Comment:
Please add some comments to explain how the ConnectionHandler field in the returned struct is used, can it be nil or must be set, etc. Or perhaps add comments to the ConnectionResponse struct to explain it there.

Reply:
Added that it MUST be set if Accept=true and added a comment to the ConnectionCallbacks interface.